### PR TITLE
Add file_name arg to download_file_command

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-31928

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml [hunk 1]
```diff
@@ -617,6 +617,12 @@ script:
         To get the ID of the file you want to download, use the msgraph-list-drive-content command.
       name: item_id
       required: true
+    - name: file_name
+      description: |-
+        The MS Graph file name.
+        To get the file name of the file you want to download, use the msgraph-list-drive-content command,
```
Comment (dorschw) [inline Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml:623]:
```suggestion
        Use msgraph-list-drive-content to retrieve the name of a file.
```

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml [hunk 2]
```diff
@@ -617,6 +617,12 @@ script:
...
+        if not provided, the file name will be item_id.
```
Comment (dorschw) [inline Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml:624]: will it be `item_id` or _the_ `item_id` value?

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml
Comment (dorschw) [inline Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml:1]: make sure to update the README as well

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_test.py [hunk 3]
```diff
@@ -173,27 +173,47 @@ def test_parse_key_to_context_exclude_keys_from_list() -> None:
...
     When
         - Using download file command in Demisto
     Then
-        - return FileResult object
+        - Ensure the `filename` is as sent in the command arguments when provided
+          otherwise, the `filename` is `item_id`
     """
     mocker.patch.object(CLIENT_MOCKER.ms_client, "http_request", return_value=response)
-    result: dict = command(CLIENT_MOCKER, args)
-    assert "Contents" in list(result.keys())
```
[the remark this answers is not in the document]
Comment (israelpoli) [reply Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_test.py:196]: yes, 
this was a trivial test for the `FileResult` function
and created a file for nothing as a result

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml (hunk 2 above)
Comment (israelpoli) [reply Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml]: item_id value
I changed

**Review by dorschw — APPROVED** [review]

**Pull request merged**